katas/README.md

```diff
@@ -5,6 +5,9 @@ A Kata is a top-level container of educational items which are used to explain a
 - Exercises: problems that the user solves by writting Q# code.
 - Questions: analytical problems that have a text answer.
 
+## Run Katas Online
```

Comment:
I took the liberty to add the new link to the Katas in the documentation, since its a small change